### PR TITLE
ZUR-153 My PR removes all import leading to the src-old folder and also all unused refs #2567 and also Fixed commit

### PR DESCRIPTION
**Pull request by sulaimon23**

### CODE REVIEW

**Comment by sulaimon23** [comment]
@JulianaSau Here is the updated push with the proper commit history

**Review by JulianaSau — CHANGES_REQUESTED** [review]
Why did you delete the `yarn.lock`?
Since it's already in the codebase, you can just let it be. Don't modify it in any way

**Comment by sulaimon23** [comment]
@JulianaSau done. Thanks

File: package.json [hunk 1]
```diff
@@ -48,6 +48,9 @@
     "prettier": "^2.3.2",
     "pretty-quick": "^3.1.1",
     "rimraf": "^3.0.2",
-    "serve": "^12.0.0"
+    "sass": "^1.56.1",
+    "sass-loader": "^13.2.0",
+    "serve": "^12.0.0",
+    "webpack": "^5.75.0"
```
Comment (JulianaSau) [inline package.json:54]: Are these dependencies you're adding related to the task you were given?

File: packages/main/webpack.config.js [hunk 2]
```diff
@@ -61,7 +61,8 @@ module.exports = (webpackConfigEnv, argv) => {
           use: "js-yaml-loader"
         },
         {
-          test: /\.css$/i,
+          test: /\.css$/,
+          include: [path.resolve(__dirname, "not_exist_path")],
```
Comment (JulianaSau) [inline packages/main/webpack.config.js:65]: Same to this

File: packages/marketplace/webpack.config.js [hunk 3]
```diff
@@ -37,7 +37,8 @@ module.exports = (webpackConfigEnv, argv) => {
     module: {
       rules: [
         {
-          test: /\.css$/i,
+          test: /\.css$/,
+          include: [path.resolve(__dirname, "not_exist_path")],
```
Comment (JulianaSau) [inline packages/marketplace/webpack.config.js:41]: Same to this

File: packages/ui/webpack.config.js [hunk 4]
```diff
@@ -42,7 +42,8 @@ module.exports = (webpackConfigEnv, argv) => {
     module: {
       rules: [
         {
-          test: /\.css$/i,
+          test: /\.css$/,
+          include: [path.resolve(__dirname, "not_exist_path")],
```
Comment (JulianaSau) [inline packages/ui/webpack.config.js:46]: Same to this

**Review by JulianaSau — CHANGES_REQUESTED** [review]
I'd suggest creating a fresh pr that isn't modifying the yarn.lock file, no changes on dependencies in the repo.
Just work on the files you were asked to change and commit only those changes

**Comment by sulaimon23** [comment]
okay thanks
@JulianaSau

**Pull request closed without merging**